server/src/main/java/io/crate/auth/AccessControlImpl.java

```diff
@@ -571,6 +572,18 @@ public Void visitAlterTableAddColumn(AnalyzedAlterTableAddColumn analysis,
             return null;
         }
 
+        @Override
```

Comment:
I will introduce a `AccessControlMayExecute` similar to `AccessControlMaySeeTest` on master, and then add test for drop column later on.